index.html

```diff
@@ -9023,6 +9041,30 @@ <h2 id="MediaStreamAudioSourceNode">
           "#control-message-queue">control message queue</a> <em>Q</em> is as
           follows.
         </p>
+        <p>
+          If the algorithm terminates normally (i.e. reaches its last step),
+          then it MUST be executed again in the future, to render the next
+          block of audio. Else, the <a>control thread</a> can make it run again
+          as needed.
+        </p>
+        <div class="note">
+          <p>
+            In practice, the <a>AudioContext</a> <a>rendering thread</a> is
+            often running off a system level audio callback, that executes in
+            an isochronous fashion. This callback passes in a buffer that has
+            to be filled with the audio that will be output. The size of the
+            buffer is often larger than a rendering quantum. In this case,
+            multiple invocations of the rendering algorithm will be called in a
+            rapid succession, in the same callback, before returning. After
+            some time, the underlying audio system will call the callback
+            again, and the algorithm will be executed again. This is an
+            implementation detail that should not be observable, apart from the
+            latency implications.
+          </p>
+          <p>
+            <a>OfflineAudioContext</a> will execute the algorithm continuously.
```

Comment:
This is in relation with the paragraph above this one, where it is explained that the audio callback mechanism forces the rendering work to happen in chunks.